src/scripts/check-platform-updates/index.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Script check platform updates
+ */
+import "dotenv/config";
+import log from "logger";
+import { prisma } from "generated/client";
+import request from "request-promise-native";
+import { size } from "lodash";
+import yargs from "yargs";
+
+exports.getPlatformReleases = getPlatformReleases;
+
+async function getPlatformReleases(uri) {
```

Comment:
```suggestion
export async function getPlatformReleases(uri) {
```

instead of `exports.getPlatformReleases`

Reply:
done.

Reply:
Did it help the export issue?

Reply:
yes